src/uct/api/uct_def.h

```diff
@@ -79,6 +79,28 @@ typedef struct uct_ep_addr       uct_ep_addr_t;
  * @}
  */
 
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Structure for scatter-gather I/O.
+ *
+ * This is special high-speed primitive to perform scatter-gather operations
```

Comment:
let's remove the "high-speed" adjective:
"Specifies a list of buffers which can be used within a single data transfer function call"